src/argilla/client/workspaces.py

```diff
@@ -110,12 +110,13 @@ def users(self) -> List["WorkspaceUserModel"]:
         Returns:
             A list of `WorkspaceUserModel` instances.
         """
+        # TODO(@alvarobartt): Maybe we should return a list of rg.User instead.
         return workspaces_api.list_workspace_users(self.__client, self.id).parsed
 
-    def __str__(self) -> str:
+    def __repr__(self) -> str:
         return (
-            f"Workspace(id={self.id}, name={self.name}, users={self.users},"
-            f" insterted_at={self.inserted_at}, updated_at={self.updated_at}"
+            f"Workspace(id={self.id}, name={self.name},"
```

Comment:
Why did we decide to remove the users from the `print`-statement? Should we keep it there for more visibility?

Reply:
For the sake of readability, I think is better to exclude them. Otherwise, the info can be hard to read. Also, we have the `workspace.users` property which will show the users correctly.